src/utilities/gql.ts

```diff
@@ -3449,6 +3460,16 @@ const gql = {
     }
   `,
 
+  UPDATE_ACTIVITY_DIRECTIVES: `#graphql
+    mutation UpdateActivityDirective($updates: [activity_directive_updates!]!) {
+      update_activity_directive_many(
```

Comment:
Can you add an `UPDATE_ACTIVITY_DIRECTIVES` to the `Queries` enum for this as well?